tests/cross_validation/test_lal_overlap.py

```
@@ -953,3 +951,137 @@ def _scatter_with_fallback(ax, x, y, c, cmap, s=20, alpha=0.8):
+# The overlap test is insensitive to any constant phase offset φ between ripple
+# and LAL, because |⟨e^{iφ}h₁|h₂⟩|² = |⟨h₁|h₂⟩|² for any real φ.  This
+# means a waveform could have a spurious +π in phifRef applied to h₊ and
+# still pass the overlap test with loss = 0.
+#
+# This test catches constant phase offsets in the POLARISATIONS by checking
+# that arg(⟨h₊_ripple | h₊_LAL⟩) ≈ 0 rad at tc = 0, phic = 0.
+#
+# Scope and limitations:
+#   - Only non-precessing waveforms: for precessing waveforms the global phase
+#     is entangled with spin azimuthal angles and requires a separate analysis.
+#   - Only h₊ is tested: for aligned-spin waveforms h× = −i cosι / antenna · h₊
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)